src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -823,6 +823,36 @@ public void testLoggedTime() throws IOException {
 
     }
 
+    @Test
+    public void testLoggedTimeInProcessFiles() throws IOException {
+        final CheckstyleAntTaskLogStub antTask = new CheckstyleAntTaskLogStub();
+        antTask.setConfig(getPath(CONFIG_FILE));
+        antTask.setProject(new Project());
+        antTask.setFile(new File(getPath(FLAWLESS_INPUT)));
+        final long startTime = System.currentTimeMillis();
+        antTask.execute();
+        final long endTime = System.currentTimeMillis();
+        final long overallLoggedTime = endTime - startTime;
+        final List<MessageLevelPair> loggedMessages = antTask.getLoggedMessages();
+
+        assertLoggedTime(loggedMessages, overallLoggedTime, "locate");
+        assertLoggedTime(loggedMessages, overallLoggedTime, "process");
```

Comment:
`testLoggedTime()` seems very similar to this new method. Is it possible to replace it with just another `assertLoggedTime(...)` line here?

Reply:
Done